### DISCUSSION TITLE
Acceptable alternate parts for the uInverter Rev C.

### DISCUSSION THREAD

**Discussion by noguchi-takahiro** [Severson-Group/AMDC-Hardware#239] [General]
Hello @elsevers @gdgugino,

Here is the part list for uInverter Rev C that we summarized on 3/22/2024.
[uInv_revC_BOM_20240322.xlsx](https://github.com/Severson-Group/AMDC-Hardware/files/14733892/uInv_revC_BOM_20240322.xlsx)

<img width="200" alt="image" src="https://github.com/Severson-Group/AMDC-Hardware/assets/114006024/846a4e66-c2c5-42ea-9598-f056923ff5dd">

I checked if each part is available in Digikey. These parts of `CP-037AH-ND` and  `A97581-ND` in the blue cells are out of stock. I was exploring Digikey to find the alternatives parts and are shown in the yellow cells and a table below.

|  Not available parts |  Alternative parts |
:--- | --- |
[CP-037AH-ND](https://www.digikey.com/en/products/filter/power-connectors/435?s=N4IgjCBcoEwBxVAYygMwIYBsDOBTANCAPZQDaIAzAGxhwUUgC6hADgC5QgDKbATgJYA7AOYgAvoRgB2GIhApIGHAWJlwMMFKkNmIdpx4CR4iSFmRyLIgHdcvAARIigwbiRsivJoQCscllAUrAGQMD5iYkA) ([Datasheet](https://www.cuidevices.com/product/resource/pj-037ah.pdf)) | [CP-037BH-ND](https://www.digikey.com/en/products/detail/cui-devices/PJ-037BH/1644548) ([Datasheet](https://www.cuidevices.com/product/resource/pj-037bh.pdf)) or [CP-037A-ND](https://www.digikey.com/en/products/detail/cui-devices/PJ-037A/1644545) ([Datasheet](https://www.cuidevices.com/product/resource/pj-037a.pdf)) |  
[A97581-ND](https://www.digikey.com/en/products/detail/te-connectivity-amp-connectors/5-1634503-1/1755969?s=N4IgTCBcDaIIIE4DsBWAHARgLQDkAiIAugL5A) ([Datasheet](https://www.te.com/commerce/DocumentDelivery/DDEController?Action=srchrtrv&DocNm=1634503&DocType=Customer+Drawing&DocLang=English))| [2111-VB334N-ND](https://www.digikey.com/en/products/detail/cinch-connectivity-solutions/VB334N/4430331) ([Datasheet](https://www.belfuse.com/resources/drawings/cinchconnectivitysolutions/vitelec/dr-ccs-vit-vb334n.pdf))|  

I am concerning about the alternative part of `CP-037AH-ND`. For this part, the rated voltage and current should be 24 V, and 5 A, respectively. Also, a mating diameter is 2.10mm ID (0.083"), 5.50mm OD (0.217"). As the alternative, `CP-037BH-ND` has the same voltage and current rating, but slightly different of the mating diameter, such as 2.50mm ID (0.098"), 5.50mm OD (0.217"). On the other hand, `CP-037A-ND` has the same mating diameter but different current rating of 2.5 A. Which do you think we should order? 

Note: red cell indicates that they are alternative parts that were already in the uInverter parts box. We can order them when the regular parts are out of stock.

**Comment by elsevers** [comment]
Thanks @noguchi-takahiro.

- I approve [CP-037A-ND](https://www.digikey.com/en/products/detail/cui-devices/PJ-037A/1644545) as an acceptable replacement for `CP-037AH-ND`. But I do not think we should use [CP-037BH-ND](https://www.digikey.com/en/products/detail/cui-devices/PJ-037BH/1644548) because of the difference in center pin size. I am not concerned about the difference in amperage rating (original connector: 5A, new connector: 2.5A). 
- I approve [2111-VB334N-ND](https://www.digikey.com/en/products/detail/cinch-connectivity-solutions/VB334N/4430331) as an acceptable replacement for [A97581-ND](https://www.digikey.com/en/products/detail/te-connectivity-amp-connectors/5-1634503-1/1755969?s=N4IgTCBcDaIIIE4DsBWAHARgLQDkAiIAugL5A). 

Let's move forward with the order.
